unblob/extractor.py

```diff
@@ -1,4 +1,6 @@
 """File extraction related functions."""
+# ruff: noqa
+
```

Comment:
No. We should adapt the code to use `pathlib` instead of straight up ignoring ruff. I can take care of it if you want.

Reply:
answered as main comment.
these changes are to be dropped.